documentation/source/user_section/command-line/sct_deepseg.rst

```diff
@@ -122,8 +122,8 @@ You can replace "``lesion_sci_t2``" with any of the task names in the table belo
 Other structures
 ----------------
 
-.. |rootlets_t2| image:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/sct_deepseg/spinal_rootlets_t2.png
-   :target: deepseg/rootlets_t2.html
+.. |rootlets| image:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/sct_deepseg/spinal_rootlets_t2.png
```

Comment:
TODO: update the figure

Reply:
I had in mind something like this, showing both T2w and MP2RAGE:

<img width="400" alt="image" src="https://github.com/user-attachments/assets/b1144a93-96ff-414d-b63b-86c1e65275d2" />

but, I guess the original figure likely looks better. I suppose we don't necessarily need to show both contrasts, as the thumbnails are intended to be informative.

<img width="400" alt="image" src="https://github.com/user-attachments/assets/aade32d0-4e06-4559-82e0-2817dc872874" />

Reply:
Because the figure is shrunken a little bit when shown in the gallery:

![image](https://github.com/user-attachments/assets/2e348e93-34a6-421e-9b3e-76d17d6da086)

I think I like the current one a little bit more? Just because the colours are still very visible in the sagittal view, whereas with the new axial-only figure it is kind of tricky to see.

>  I suppose we don't necessarily need to show both contrasts, as the thumbnails are intended to be informative.

I think you're right, that maybe trying to fit a lot of information (extra slice, extra contrast) might make it harder to read?

Reply:
agree. so what about keeping the current figure and just updating the caption to `T2/MP2RAGE`?

<img width="400" alt="image" src="https://github.com/user-attachments/assets/89eba661-8082-45fd-acdc-faebd10ea184" />

Reply:
Perfect!

Reply:
Thanks! Updated in https://github.com/spinalcordtoolbox/doc-figures/commit/bc2af9479274c75b0d4d8647fc8eda5202413de0